libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationViewContext.kt

```diff
@@ -26,3 +32,11 @@ internal class DropInNavigationViewContext(
         .withAboveLayerId(RouteLayerConstants.TOP_LEVEL_ROUTE_LINE_LAYER_ID)
         .build()
 }
+
+/**
+ * Helper extension to map [UIBinder] inside a [UICoordinator].
+ * Uses a distinct by class to prevent refreshing views of the same type of [UIBinder].
+ */
+internal fun <T : UIBinder> DropInNavigationViewContext.flowUiBinder(
+    mapper: suspend (value: NavigationUIBinders) -> T
+): Flow<T> = this.uiBinders.map(mapper).distinctUntilChangedBy { it.javaClass }
```

Comment:
I do this knowing that, new instances will not change the view. It needs to be a different class type.

I'm considering this better than, requiring all `UIBinder` implementations to override equals/hashcode.